pingpong/ud_pingpong.c

```diff
@@ -341,7 +341,7 @@ static int common_setup(void)
 	if (ret)
 		return ret;
 
-	ret = fi_getinfo(FT_FIVERSION, node, service, flags, hints, &fi);
+	ret = fi_getinfo(FI_VERSION(1, 1), node, service, flags, hints, &fi);
```

Comment:
We may need this to become a configure setting or something, so that we can easily test that libfabric supports any older applications.